### PR TITLE
Call the backend endpoint to update document with import_id

### PR DESCRIPTION
**Pull request by diversemix**

### CODE REVIEW

File: src/navigator_data_ingest/base/actions.py [hunk 1]
```diff
@@ -51,9 +55,11 @@ def handle_all_documents(
     The remote filename follows the template on
     https://www.notion.so/climatepolicyradar/Document-names-on-S3-6f3cd748c96141d3b714a95b42842aeb
     """
+    token = get_machine_user_token()
     tasks = {
         executor.submit(
             _handle_document,
+            token,
```
Comment (joel-wright) [inline src/navigator_data_ingest/base/actions.py:62]: I can't remember how long our access tokens are valid for, hopefully these tokens won't expire if we upload lots of documents!

File: src/navigator_data_ingest/base/actions.py [hunk 2]
```diff
@@ -146,14 +153,22 @@ def _handle_document(
...
+        )
+        # FIXME: Remove the debug and test with corresponding backend change
+        print("-" * 100)
+
+        if response.status_code >= 300:
+            # TODO: More nuanced status response handling
+            _LOGGER.error(
+                f"Failed to update entry in the database for '{document.import_id}': "
+                f"[{response.status_code}] {response.text}"
+            )
```
Comment (joel-wright) [inline src/navigator_data_ingest/base/actions.py:170]: is this currently highlighted because we always fail?

**Review by joel-wright — APPROVED** [review]

File: src/navigator_data_ingest/base/actions.py (hunk 1 above)
Comment (diversemix) [reply src/navigator_data_ingest/base/actions.py:62]: Ooo - good point to raise :thinking:

File: src/navigator_data_ingest/base/actions.py (hunk 2 above)
Comment (diversemix) [reply src/navigator_data_ingest/base/actions.py:170]: What do you mean by "highlighted" ?

Comment (joel-wright) [reply src/navigator_data_ingest/base/actions.py:170]: I just meant these extra long print/log separators around the code blocks

**Review by joel-wright — APPROVED** [review]
lgtm

**Pull request merged**